Python/bytecodes.c

```diff
@@ -720,7 +720,11 @@ dummy_func(
                 exc = args[0];
                 /* fall through */
             case 0:
-                ERROR_IF(do_raise(tstate, exc, cause), exception_unwind);
+                if (do_raise(tstate, exc, cause)) {
```

Comment:
I'm confused about error reporting from do_raise. If there is no exception set, it sets an exception and returns 0. What does that mean?

Reply:
`do_raise` returns 0 if it raises, and 1 if it **re**raises.
Historical reasons probably :shrug: